README.md

```diff
@@ -70,15 +70,58 @@ export default function App({ Component, pageProps }: AppProps) {
 
 Then, in your register, login and logout forms you can use the auth methods (`signIn`, `signOut`, `isAuthenticatin`) provided by the `useSaleorAuthContext()`. For example, `signIn` is usually triggered when submitting the login form credentials.
 
-```ts
-const { signIn, signOut, isAuthenticating } = useSaleorAuthContext();
-```
+```tsx
+import React, { FormEvent } from "react";
+import { useSaleorAuthContext } from "@saleor/auth-sdk/react";
+import { useQuery } from "@apollo/client";
+import { CurrentUserDocument } from "../generated/graphql";
 
-```ts
-const response = await signIn({
-  email: "example@mail.com",
-  passowrd: "password",
-});
+export const LoginPage = () => {
+  const { signIn, signOut, isAuthenticating } = useSaleorAuthContext();
+
+  const [{ data: currentUser, loading: isLoadingCurrentUser }] = useQuery(CurrentUserDocument);
+
+  // important: wait for both the end of auth process and the end of query invocation
+  const isLoading = isAuthenticating || isLoadingCurrentUser;
```

Comment:
Are you sure it's necessary to wait for `isAuthenticating`? I believe it's implicit and we can simplify these examples.

Reply:
Why? It doesn't affect the way this example works – it'd be the same with or without `isAuthenticating`. IMO it's making the example more complicated than it needs to be.

Reply:
In other words: `isAuthenticating || isLoadingCurrentUser === isLoadingCurrentUser`